index.html

```diff
@@ -189,6 +224,7 @@ <h4>Skills</h4>
 
     <form class="last_section" action="https://formspree.io/f/xbjwkevl" method="post">
```

Comment:
- [ ] You are almost there! Let's center the submit button and fix the problem with the background in order to display it correctly like in the Figma template:

Template:
![Captura de pantalla 2022-03-26 a la(s) 11 04 52 a  m](https://user-images.githubusercontent.com/83717285/160248332-1958734e-6e3b-44a7-bf4d-371f2edbee5c.png)

Your design:
![Captura de pantalla 2022-03-26 a la(s) 11 05 13 a  m](https://user-images.githubusercontent.com/83717285/160248342-53c4b323-0d39-4350-bf53-a4aa4395a08d.png)